product_docs/docs/postgres_distributed_for_kubernetes/1/rel_notes/1_0_rel_notes.mdx

```diff
@@ -1,28 +1,36 @@
 ---
-title: 'EDB Postgres Distributed for Kubernetes 1.0 release notes'
-navTitle: "Version 1.0"
+title: 'EDB Postgres Distributed for Kubernetes 1.0.0 release notes'
+navTitle: "Version 1.0.0"
 ---
 
-**Release date:** 15 February 2024
-
-This is the first major stable release of EDB Postgres Distributed for
-Kubernetes, a Kubernetes operator to deploy and manage
-EDB Postgres Distributed clusters.
-
-The operator implements the `PGDGroup` custom resource
-in the API group `pgd.k8s.enterprisedb.io`.
-This resource can be used to create and manage EDB Postgres Distributed clusters
-inside Kubernetes with capabilities including:
-
--   Deployment of EDB Postgres Distributed clusters with versions 5 and later
--   Additional self-healing capability on top of that of Postgres Distributed,
-    such as recovery and restart of failed PGD nodes
--   Definition of the services to connect applications to the
-    write leader of each PGD group
--   Implementation of Raft subgroups
--   Support for Local Persistent Volumes with PVC templates
--   Reuse of Persistent Volumes storage in Pods
--   TLS connections and client certificate authentication
--   Continuous backup to an S3 compatible object store
--   Pause and resume a PGD cluster, saving computational resources by temporarily
-    removing database pods while keeping the database PVCs.
+Released: 24 Apr 2024
+
+This is the first major stable release of EDB Postgres Distributed for Kubernetes, a Kubernetes operator to deploy 
+and manage EDB Postgres Distributed clusters.
+
+## Highlights of EDB Postgres Distributed for Kubernetes 1.0.0
+
+The operator implements the `PGDGroup` custom resource in the API group `pgd.k8s.enterprisedb.io`. You can use this resource 
+to create and manage EDB Postgres Distributed clusters inside Kubernetes with capabilities including:
+
+* Deployment of EDB Postgres Distributed clusters with versions 5 and later.
+* Additional self-healing capability on top of that of Postgres Distributed, such as recovery and restart of failed PGD nodes.
+* Definition of the services to connect applications to the write leader of each PGD group.
+
+!!! Note
+The EDB Postgres Distributed for Kubernetes operator leverages 
+[EDB Postgres for Kubernetes](https://www.enterprisedb.com/docs/postgres_for_kubernetes/latest/) (PG4K) and inherits many 
+of that project's capabilities. EDB Postgres Distributed for Kubernetes version 1.0.0 is based, specifically, on release 1.22 of PG4K. 
+Please refer to the [PG4K release notes](https://www.enterprisedb.com/docs/postgres_for_kubernetes/latest/rel_notes/) for more details.
+!!!
```

Comment:
Not sure what tooling you guys use for warning boxes and the like.
For MkDocs, which CNP dev team uses, the content in the note needs to be indented 4 spaces (lines 21 - 24 in this PR)

Reply:
We use proper admonitions... we can indent it upstreaming easily enough, but as always I will warn of the dangers of unfenced admonitions (and their weird effects on subsequent text).